docker/compose-controller-spark-sql-single.yaml

```diff
@@ -72,6 +72,9 @@ services:
     container_name: spark-thriftserver
     command:
       - sbin/start-thriftserver.sh
+      - --driver-memory
+      # You may need to increase this if your queries/data are too large.
+      - 5g
```

Comment:
I assume this is the value for `driver-memory`, does it need to be on a separate line?

Reply:
Yes, I think compose supplies each element in this array as a single argument, so if I put this on the previous line, `start-thriftserver.sh` fails by saying that is does not understand "--driver-memory 5g".